guides/testing-other-projects.md

```diff
@@ -0,0 +1,27 @@
+# Testing other projects
+
+In CI, the Cypress binary and npm package are built and uploaded to `cdn.cypress.io`. Then, [`/scripts/test-other-projects.js`](../scripts/test-other-projects.js) is run.
+
+This script creates commits inside of several test projects in order to trigger a realistic, continous-integration test of Cypress.
+
+For a list of the projects, see the definition of `_PROVIDERS` in [`/scripts/binary/bump.js`](../scripts/binary/bump.js).
```

Comment:
1 - looks like I missed a place I needed to update with AppVeyor...Will have to do that!
2 - Looking at [the list in bump.js](https://github.com/cypress-io/cypress/blob/develop/scripts/binary/bump.js#L24], this is only when merged into develop correct?

CI also clones and test the against these projects...where do those come to play in this doc? Might be worth adding info that describe two test strategies
- [cypress-example-kitchensink](https://github.com/cypress-io/cypress/blob/develop/circle.yml#L1610)
- [cypress-example-todomvc](https://github.com/cypress-io/cypress/blob/develop/circle.yml#L1936)
- [cypress-example-api-testing](https://github.com/cypress-io/cypress/blob/develop/circle.yml#L1954)
- cypress-example-piechopper
- cypress-test-awesome-typescript-loader
- cypress-example-conduit-app
- cypress-example-recipes
- cypress-realworld-app
- a scaffolded Typescript Project

Reply:
It is develop by default, but you can add a branch for all of these extra tests to run against also if you update the circle.yml

Reply:
Added a note about these in c8d16320bc.